sphinx/source/docs/user_guide/interaction/callbacks.rst

```diff
@@ -77,10 +77,57 @@ is executed to update some data:
     :source-position: above
 
 
+
+CustomJS for Event classes
```

Comment:
actually - you'll have to divide the above section into "CustomJS for Generic Events" and "CustomJS for Model Property Change Events"